src/lib/tooltip/tooltip.ts

```diff
@@ -461,4 +463,13 @@ export class TooltipComponent {
       this.hide(0);
     }
   }
+
+  /**
+   * Ensures that the tooltip text has rendered. Mainly used for rendering the initial text
+   * before positioning a tooltip, which can be problematic in components with OnPush change
+   * detection.
+   */
+  _updateView(): void {
+    this._changeDetectorRef.detectChanges();
```

Comment:
Hmm, I thought I tried it and it wasn't working. Looking at it again, it seems like it's the same. I'll change it.